### PR TITLE
use valid XML in code block

### PR DESCRIPTION
**Pull request by xabbuh**

### CODE REVIEW

**Comment by xabbuh** [comment]
Looks like this is related to a recent update of Sphinx on Travis. [1.3.4 didn't fail](https://travis-ci.org/symfony/symfony-docs/builds/103688233) with this while [1.3.5 now does](https://travis-ci.org/symfony/symfony-docs/builds/104404503).

**Comment by WouterJ** [comment]
unfortunately, it still breaks.

**Comment by WouterJ** [comment]
@xabbuh since 1.3.5, pygments errors are now triggered as warnings in Sphinx. Pygments always had an issue rendering this code blocks, it seems.

The problem is the `&` char, which has to be `&amp;`: http://pygments.org/demo/3557987/

**Comment by xabbuh** [comment]
Yeah, there are some more issues in other files. I'll fix them and make a big update here.

File: cookbook/doctrine/mongodb_session_storage.rst [hunk 1]
```diff
@@ -162,7 +162,7 @@ Because MongoDB uses dynamic collection schemas, you do not need to do anything
 session collection. However, you may want to add an index to improve garbage collection performance.
 From the `MongoDB shell`_:
 
-.. code-block:: sql
+.. code-block:: text
```
Comment (WouterJ) [inline cookbook/doctrine/mongodb_session_storage.rst]: that's weird, SQL should be available.

Comment (WouterJ) [inline cookbook/doctrine/mongodb_session_storage.rst]: Sorry, my bad, it isn't sql.

Comment (WouterJ) [inline cookbook/doctrine/mongodb_session_storage.rst]: as it's shell code, shouldn't we use bash + `$` though?

Comment (xabbuh) [inline cookbook/doctrine/mongodb_session_storage.rst]: It even isn't shell code, is it? I mean it's the MongoDB shell, but I am not sure if the bash highligher will handle that properly. Let's try it (see #6192).

**Pull request merged**